### ISSUE TITLE
Sample code snippet error

### ISSUE THREAD

**Issue by jithenece** [MHubAI/SegDB#6]
I tried to run the example from README.md and got exception

```
from segdb.tools import DcmqiDsegConfigGenerator

# generate json meta generator instance
generator = DcmqiDsegConfigGenerator(
    model_name = 'My Model',
    body_part_examined = 'CHEST'
)

# add segmentation
generator.addItem(
    file = 'path/to/segmentation.nii.gz',
    segment_ids = 'HEART',
    model_name = 'My Model'
)

# save json
generator.save('path/to/meta.json')
```

Error details below
`    raise Exception("Invalid segment ids: " + ', '.join(item.malformed_segment_ids or []))
Exception: Invalid segment ids: H, E, A, R, T
`

I tried on python3.10 and python3.12

**Comment by jithenece** [comment]
segment_ids needs to be passed ['HEART'] instead of 'HEART'

**Comment by LennyN95** [comment]
You're right, segdb IDs need to be passed as a list of strings into `segment_ids`. The example code is outdated and I will fix this. Thank you @jithenece for taking the time and opening this issue!

**Comment by LennyN95** [comment]
Fixed in f165a6c7d797e043530e576225b79a4ad3cf08d6.
